frontend/src/static/js/v_authorship.js

```diff
@@ -32,6 +32,8 @@ window.vAuthorship = {
     return {
       isLoaded: false,
       files: [],
+      fileLineCountArray: [],
+      fileLineCount: "",
```

Comment:
Count generally denotes an integer variable or a value. 
Perhaps `filesLineInfo` will be more accurate?